isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -24,30 +24,132 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
+    If you are a developer, we refer you to our <a href="https://github.com/USGS-Astrogeology/ISIS3">GitHub repository.</a>
+  </p>
+
+
+  <h3>ISIS3 Installation With Conda</h3>
+    <ol>
+      <li>Download either the Anaconda or Miniconda installation script for your OS platform.  Anaconda is a much larger distribtion of packages supporting scientific python, while Miniconda is a minimal installation and not as large: 
+      <a href="https://www.anaconda.com/download">Anaconda installer</a>, <a href="https://conda.io/miniconda.html">Miniconda installer</a></li>
+      <li>Open a terminal window in the directory where you downloaded the script, and run the following commands.  In this example, we chose to do a full install of Anaconda, and our OS is Linux-based.  Your file name may be different depending on your environment.</li>
+        <pre>
+          chmod +x Anaconda3-5.2.0-Linux-x86_64.sh
+          ./Anaconda3-5.2.0-Linux-x86_64.sh          
+        </pre>
+        This will start the Anaconda installer which will guide you through the installation process.
+      <li>After the installation has finished, open up a bash prompt in your terminal window.</li>
+      <li>Create a new environment for your ISIS3 installation:
+        <pre>
+          #Add the following channels to the Anaconda installation
+          conda config --add channels conda-forge
+          conda config --add channels usgs-astrogeology
+
+          #Create a new conda environment to install ISIS3 in
+          conda create -n isis3
+
+          #Activate the environment
+          source activate isis3
+
+          #Download the ISIS3 version with label alpha
+          conda install -c usgs-astrogeology isis3
+          
+          #Execute the ISIS3 variable initialization script.  
+          #This script prepares default values for:  $ISISROOT/$ISIS3DATA/$ISIS3TESTDATA
+
+         ./$CONDA_PREFIX/scripts/isis3VarInit
+
+          #Run the source activate command a second time to process the changes:
+
+          source activate isis3alpha
+
+
+
+
+
+
+        </pre>
+      </li>
+    </ol>
   <h3>Operating System Requirements</h3>
-  <p>ISIS runs on many UNIX variants. ISIS does not run on MS Windows. The UNIX variants supported are listed here:</p>
+  <p>ISIS3 runs on many UNIX variants. ISIS does not run natively on MS Windows,
+     although it has been successfully run on Windows 10 using the Windows
+     Subsystem for Linux (WSL).  Instructions for doing this can be found
+     <a href="#RunningOnWindows">here.</a>
+
+    The UNIX variants ISIS3 has been successfully built on are:</p>
     <ul>
       <li>Ubuntu 18.04 LTS</li>
       <li>Mac OS X 10.13.6 High Sierra</li>
       <li>CentOS Linux 7.2.1511</li>
       <li>Fedora 28</li>
     </ul>
 
+
+    <p>ISIS3 is also built and tested nightly on the following operating systems.</p>
+
+    <ul>
+      <li>Ubuntu 14.04.3 LTS</li>
```

Comment:
Is this true any more?

Reply:
Prog22 runs Ubuntu 14.04.3 and we're running tests on it nightly.

Reply:
Never mind...just talked with Ian and we're not supporting Ubuntu 14.04 in the future so I'll delete the reference to it.

Reply:
I would just remove this entire section ("ISIS3 is also built and tested nightly on the following operating systems"), since they are listed above (ubuntu18, macOS 10.13, CentOS 7, Fedora28)